test/reference_image_tests/simple.jl

```diff
@@ -0,0 +1,208 @@
+using MakieGallery: @substep
+
+@block SimonDanisch ["tests"] begin
+    ## Scatter
+    @cell begin
```

Comment:
would multiple cells be better here?  all of these in one cell makes it kind of hard to debug.

Reply:
Also, we do need to separate 2d examples from 3d ones (for CairoMakie)

Reply:
Yeah I need to figure out how to do this... I had to put them in one cell, to have theglobally shared functions and data work!

Reply:
You could also put toplevel code in, that works now